src/DBHandler.py

```diff
@@ -15,182 +15,209 @@
 # You should have received a copy of the GNU General Public License
 # along with Bolinho.  If not, see <http://www.gnu.org/licenses/>.
 
-import orm_sqlite
-from datetime import date, time
-from os.path import exists as file_exists
+from peewee import *
+import datetime
+from os.path import exists as path_exists
 
 
-class Material(orm_sqlite.Model):
-    id = orm_sqlite.IntegerField(primary_key=True)
-    name = orm_sqlite.StringField()
-    batch = orm_sqlite.StringField()
+db_path = "bolinho.db"
```

Comment:
Eu to usando uma pasta chamada "persist" pra guardar informação persistente, acho que seria bom colocar o db lá